MultiPointRasterGM.h

```diff
@@ -0,0 +1,295 @@
+//##############################################################################
+//
+//    FILENAME:          MultiPointRasterGM.h
+//
+//    CLASSIFICATION:    Unclassified
+//
+//    DESCRIPTION:
+//
+//    
+//
+//    LIMITATIONS:       None
+//
+//
+//    SOFTWARE HISTORY:
+//     Date          Author   Comment
+//     -----------   ------   -------
+//     31-Dec-2021   EJR    Initial version.
+//
+//    NOTES:
+//
+//##############################################################################
+
+#ifndef _H_MULTI_RASTER_GM_H
+#define _H_MULTI_RASTER_GM_H
+
+#include "csmMultiPoint.h"
+#include "RasterGM.h"
+
+#define CSM_MPR_FAMILY "MultiPoint"
+
+namespace csm
+{
+class CSM_EXPORT_API MultiPointRasterGM : public RasterGM
```

Comment:
I don't think that `MultiPointRasterGM` should derive from `RasterGM`.  I think it should be its own interface that models could implement by deriving from both `RasterGM` and `MultiPointRasterGM`.  That way it is optional, and clients can determine if it has this interface by doing something like `MultiPointRasterGM* mpModel = dynamic_cast<MultiPointRasterGM*>(someRasterModel); if (mpModel) { ... }`

Doing this allows for other classes like `MPRGWrapper` to only need to implement `MultiPointRasterGM` methods and not pass-throughs for all of the `RasterGM` methods.

Reply:
multiple designs were considered including this one.